hello-signing.md

```diff
@@ -0,0 +1,129 @@
+# Hello World for Notation
+
+To get started with Notation, the most basic steps involve:
+
+1. Acquiring the Notation CLI
+2. Generating a Private Key
+3. Signing an Artifact
+4. Verifying an Artifact with a Notary Signature
+
+This doc outlines progressive scenarios enabling a range of scenarios from the first use to a production deployment.
+
+## Scenarios
+
+To demonstrate how to store and sign a set of supply chain artifacts, we'll walk through a set of scenarios:
+- Singing a single container image
+- Publishing the container image across public registries, 
+- Importing public content to a private registry
+- Promoting from dev through production
+
+![](../media/notary-e2e-scenarios.svg)
+
+- Wabbit Networks is a small software company that produces network monitoring software.
+- ACME Rockets wishes to acquire network monitoring software.
+- ACME Rockets doesn't know about Wabbit Networks, but finds their [net-monitor software in Docker Hub](https://hub.docker.com/r/wabbitnetworks/net-monitor)
+- Since they've never heard of Wabbit Networks, they're a little reluctant to run network software without some validations.
+- ACME Rockets has a policy to only import Docker Hub certified software, or other approved vendors.
+- Wabbit Networks works with Docker Hub to get certified, to help with their customer confidence.
+- ACME Rockets will only deploy software that's been scanned and approved by the ACME Rockets security team. They know it's been approved because all approved software has been signed by the ACME Rockets security team.
+
+## Getting Started
+- Setup a few environment variables.  
+  >Note see [Simulating a Registry DNS Name](#simulating-a-registry-dns-name) to use `registry.wabbit-networks.io`
+  ```bash
+  export PORT=5000
+  export REGISTRY=localhost:${PORT}
+  export REPO=${REGISTRY}/net-monitor
+  export IMAGE=${REPO}:v1
+  ```
+- Install [Docker Desktop](https://www.docker.com/products/docker-desktop) for local docker operations
+- Run a local instance of the [CNCF Distribution Registry][cncf-distribution]
+  ```bash
+  docker run -d -p ${PORT}:5000 ghcr.io/oras-project/registry:latest
+  ```
+- Acquire the Notation CLI  
+Notation releases can be found at: [Notation Releases][notation-releases]  
+  ```bash
+  curl -O ~/bin/notation https://github.com/notaryproject/notation/releases/download/v0.0.1-alpha/notation-v0.0.1-alpha.1-linux-amd64
```

Comment:
umm, oss here :)

Reply:
bit.ly then? :D

Reply:
_Just to elaborate on my concern/opinion._ 

I have left a lot of comments on broken and outdated links in github documentation. I know it's not up to us to solve the problem. But I think it's in the spirit of OSS if we should ensure working links for getting-started docs like these. 

Aka.ms might be internally allocated, but in this case, I feel it is at the benefit of OSS. An aka.ms link communicates several things:
a) Someone at Microsoft has chosen this link
b) We can find who made this link in case it goes stale
c) We can repair the link without needing to change the repo
d) We're committed to this project by offering these resources, as someone will remain responsible for this link internally

There is precedence for this in OSS. For example, when you install tools to work with GRPC in go: 

```
$ go install **google.golang.org/protobuf/cmd/**protoc-gen-go@v1.26
$ go install **google.golang.org/grpc/cmd/**protoc-gen-go-grpc@v1.1
```

(That being said, it would be nice if we did something similar for `oras`)

Reply:
I completely appreciate the stale issue. If it were an aka.ms link, than only Microsoft folks can edit them. In this case, anyone can make a PR to merge the changes. 
To put this PR into perspective, this is a doc we're using to stabilize the cli and put some scenarios around what/why we're building out the specific CLIs. Nate is building the website through [notaryproject.dev](https://github.com/notaryproject/notaryproject.dev)

I don't expect any of this to be final, and we'll iterate. Particularly the specific releases.
I'm hoping we'll land something more dynamic, or point folks to: https://github.com/notaryproject/notation/releases